### DISCUSSION TITLE
Install phase

### DISCUSSION THREAD

**Discussion by pjljvandelaar** [alire-project/alire#995] [Ideas]
The [package manager of Haskell](https://www.haskell.org/cabal/)
has an explicit install phase: see e.g. https://cabal.readthedocs.io/en/3.6/cabal-commands.html#cabal-v2-install:
```
cabal v2-install [FLAGS] PACKAGES builds the specified packages and symlinks/copies their executables in installdir (usually ~/.cabal/bin).
```

Should alire also have this phase?
Such that when users put this `installdir` on their path, they can run all programs installed by alire directly!

P.S. I have work quite a lot with cabal and I liked it a lot.
I think my experience with cabal trigger this idea (see also question 2 in https://github.com/AdaCore/libadalang-tools/issues/21#issuecomment-1099945322)

**Comment by Fabien-Chouteau** [comment]
Hi @pjljvandelaar,

This is something we are working on: https://github.com/alire-project/alire/pull/948

**Comment by pjljvandelaar** [comment]
Don't know how, but this discussion can be closed since indeed you are already working on it!

**Comment by Fabien-Chouteau** [comment]
Doesn't look like GitHub Discussions can be closed
